app/konnect/getting-started/configure-mesh.md

```diff
@@ -0,0 +1,106 @@
+
+For the purposes of this guide, you'll use {{site.mesh_product_name}} as your service mesh to create a global control plane. You will use Kubernetes to install various service mesh components.
+
+## Prerequisites
+
+* [A Kubernetes cluster with a load balancer](https://kubernetes.io/docs/setup/)
+* [`kubectl` installed and configured to communicate with your Kubernetes cluster](https://kubernetes.io/docs/tasks/tools/#kubectl)
+* [Download the latest version of {{site.mesh_product_name}}](/mesh/latest/production/install-kumactl/)
+
+## Create a global control plane in {{site.konnect_short_name}}
+ 
+1. From the left navigation menu in {{site.konnect_short_name}}, open {% konnect_icon mesh-manager %} [**Mesh Manager**](https://cloud.konghq.com/mesh-manager).
+1. Click **New Control Plane**.
+1. Enter "example-cp" in the **Name** field.
+1. Click **Save**.
+
+You now have a {{site.mesh_product_name}} global control plane. This control plane can't do anything at the moment because we need to connect a zone to it next. 
+
+## Create a zone in the global control plane
+
+After creating the global control plane, you must add a zone to that control plane. Adding a zone allows you to manage services added to that zone and send and receive configuration changes to the zone. 
+
+1. In {% konnect_icon mesh-manager %} [**Mesh Manager**](https://cloud.konghq.com/mesh-manager), click the `example-cp` control plane you just created, and then click **Zones** in the sidebar.
+1. Click **Create Zone**. 
+1. Enter "eu" in the **Name** field for the new zone, and then click **Create Zone & generate token**. 
+    {:.note}
+    > **Note:** The zone name must consist of lower case alphanumeric characters, `.`, or `-`. It must also start and end with an alphanumeric character.
+1. Ensure both [**Zone Ingress**](/mesh/latest/production/cp-deployment/zone-ingress/) and [**Zone Egress**](/mesh/latest/production/cp-deployment/zoneegress/) are enabled.
+1. Follow the instructions that display to set up Helm and a secret token. 
+    Once {{site.konnect_short_name}} finds the newly created zone, it will display it. 
+1. Click **Next**.
+
+You now have a very basic {{site.mesh_product_name}} service mesh added to {{site.konnect_short_name}}. This service mesh can't do anything at the moment until we add services and additional configurations to it.
+
+## Configure `kumactl` to connect to your global control plane
+
+`kumactl` is a CLI tool that you can use to access {{site.mesh_product_name}}. It can create, read, update, and delete resources in {{site.mesh_product_name}} in Universal/{{site.konnect_short_name}} mode.
+<!--* (not sure if I removed the right line, so keeping in for easy swap 
+Perform read-only operations on {{site.mesh_product_name}} resources on Kubernetes. -->
+
+You connect `kumactl` to the global control plane in {{site.konnect_short_name}} so that you can run commands against the control plane.
```

Comment:
There's now a button with these instructions straight in the GUI should we just tell people to go there and follow the instructions?